docs/releases/2.2.txt

```diff
@@ -43,7 +43,9 @@ Minor features
 :mod:`django.contrib.admin`
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-* ...
+* Use the new ``django.contrib.admin.EmptyFieldListFilter`` in
```

Comment:
I think the wording should be along the lines of ``Added :class:`django.contrib.admin.EmptyFieldListFilter` to allow filtering of objects with an empty or not field value...`` I'd also mention that it abstracts the `null`/`blank` notion from the user,